src/benchmark_run.py

```diff
@@ -187,6 +187,7 @@ def __init__(self,
                  injectors=None,
                  java=None,
                  jar=None,
+                 tag=None,
```

Comment:
Should avoid making changes to the core in `dev-gui` without them being pushed to `dev`.

Reply:
#148 addresses this.